### PR TITLE
Synchronize Dockerfiles for old upstream and Prow

### PR DESCRIPTION
**Pull request by KevinFCormier**
Signed-off-by: Kevin Cormier <kcormier@redhat.com>

open-cluster-management/backlog#13008

### CODE REVIEW

File: Dockerfile.prow [hunk 1]
```diff
@@ -6,14 +6,39 @@ WORKDIR /opt/app-root/src/application-ui
 COPY . /opt/app-root/src/application-ui/
 RUN yum install git -y
```
Comment (KevinFCormier) [inline Dockerfile.prow:7]: @chenz4027 Do you know why we would need git installed? I noticed the same thing in console-api actually.

Comment (KevinFCormier) [reply Dockerfile.prow]: I had to fix a typo so I have tried removing it in my next commit.

Comment (chenz4027) [reply Dockerfile.prow]: I remember that it was somehow a dependency when I started working on it, but maybe it's not required anymore. If it builds okay here I think it's pretty safe to remove it: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/open-cluster-management_application-ui/1263/pull-ci-open-cluster-management-application-ui-main-images/1400557386556510208

**Review by chenz4027 — APPROVED** [review]
Looks good. Thanks Kevin!

**Pull request merged**